Dockerfile

```diff
@@ -36,7 +36,8 @@ RUN dnf install -y python3 python3-requests && \
     dnf install -y python3-gunicorn openstack-ironic-api openstack-ironic-conductor crudini \
         iproute dnsmasq httpd qemu-img iscsi-initiator-utils parted gdisk psmisc \
         mariadb-server genisoimage python3-ironic-prometheus-exporter \
-        python3-jinja2 python3-sushy-oem-idrac python3-ibmcclient && \
+        python3-jinja2 python3-sushy-oem-idrac python3-ibmcclient \
+        python3-bcrypt && \
```

Comment:
This should be pulled in by the ironic-lib packaging change https://review.rdoproject.org/r/#/q/Ic8e6cead9f48af730e125fed78eb4fcbfcd9243e

Reply:
oh god to know Steve!